sound/soc/sof/pm.c

```diff
@@ -80,11 +80,18 @@ static int sof_resume(struct device *dev, bool runtime_resume)
 
 	/*
 	 * Nothing further to be done for platforms that support the low power
-	 * D0 substate.
+	 * D0 substate, and we need to re-enable the DMA trace if it was stopped
 	 */
 	if (!runtime_resume && sof_ops(sdev)->set_power_state &&
-	    old_state == SOF_DSP_PM_D0)
+	    old_state == SOF_DSP_PM_D0) {
+		ret = snd_sof_enable_trace(sdev);
+		if (ret < 0)
+			/* non fatal */
+			dev_warn(sdev->dev,
+				 "failed to enable trace after resume %d\n",
+				 ret);
```

Comment:
one line or maybe 2 lines?

Reply:
It will exceed 80 word/line. I checked the pm.c, and found this file always strictly follows 80w/l in other places.

Reply:
the limit is 100 chars

Reply:
I believe in new code you can use upto 100 chars and I think here it makes sense.